src/v0/destinations/braze/transform.js

```diff
@@ -320,6 +321,38 @@ function processTrackEvent(messageType, message, destination, mappingJson) {
   payload = addMandatoryEventProperties(payload, message);
   payload.properties = properties;
 
+  try {
+    // add,update,remove
+    if (destination.Config.enableNestedArrayOperations) {
+      Object.keys(properties)
+        .filter(k => Array.isArray(properties[`${k}`]))
+        .forEach(key => {
+          // if not specified, send as create attribute
+          if (!properties.nestedOperationType) {
+            attributePayload[`${key}`] = properties[`${key}`];
```

Comment:
Can we do this instead ?
```suggestion
attrtibute[key] = properties[key]
```